judge/models/contest.py

```diff
@@ -81,6 +81,10 @@ class Contest(models.Model):
     registration_end_time = models.DateTimeField(verbose_name=_('registration end time'),
                                                  help_text=_('Allow registration until the specified time.'),
                                                  blank=True, null=True)
+    partially_hide_scoreboard = models.BooleanField(verbose_name=_('Partially hide scoreboard'),
+                                                    help_text=_('Whether the scoreboard should be shown hidden '
+                                                    'until the member\'s contest window is over.'),
```

Comment:
indent to match the start of the previous quote (see other help texts for reference)

Reply:
Done